test/snapshots/test/custom-serializers.test.ts

```diff
@@ -6,6 +6,6 @@ test('it should pass', async () => {
     root: 'test/fixtures/custom-serializers',
   })
 
-  expect(stdout).toContain('✓ custom-serializers.test.ts >')
+  expect(stdout).toContain('✓ custom-serializers.test.ts')
```

Comment:
When running it without `CI=true`, it looks like `✓ custom-serializers.test.ts (1)` so it was failing locally. I just stripped `>` at the end so both works.